kubetest2-ec2/config/configure.sh

```diff
@@ -152,6 +152,8 @@ else
     --retry-delay 10 "${TARBALL_GCS_PATH}")
   tar xvf "${TARBALL}"
   rm -f "${TARBALL}"
+  # copy `runc` in ${CONTAINERD_HOME} to a directory in PATH as well
+  cp usr/local/sbin/runc /bin/runc || true
```

Comment:
Can you add a link to the containerd PR or issue and mention this is only needed until we use a fixed version?

Reply:
Done!

Reply:
Did you forget to push?